amp.php

```diff
@@ -387,6 +387,10 @@ function amp_init() {
 	}
 
 	if ( AMP_Options_Manager::is_stories_experience_enabled() ) {
+		AMP_Story_Legacy_Post_Type::register();
+	}
+
+	if ( AMP_Options_Manager::is_stories_editor_enabled() ) {
```

Comment:
We discussed over the call of keeping just one checkbox for Stories -- let's keep just one as discussed, if it's enabled, both are displayed, if it's disabled, none is displayed.

Reply:
Sorry, I didn't leave a note here. I tried to implement that, but couldn't. The other editor required the latest version of gutenberg. We shouldn't lock down our new editor with this requirement.